docs/scos/dev/front-end-development/migration-guide-switch-from-tslint-to-eslint.md

```diff
@@ -0,0 +1,139 @@
+
+## Overview
+
+TSLint support was discontinued as of December 1, 2020. The solution is migration to ESLint as the default linter for TypeScript and JavaScript.
+
+*Estimated migration time: 1h*
+
+## 1) Update dependencies
+
+1. In `package.json`: 
+    - Add or update the following dependencies to the new version:
+
+      ```json
+      {
+          "@spryker/frontend-config.eslint": "~0.1.0",
+          "@typescript-eslint/eslint-plugin": "~5.54.0",
+          "@typescript-eslint/parser": "~5.54.0",
+          "eslint": "~8.35.0",
+          "prettier": "~2.8.4"
+      }
+      ```
+
+    - Remove the following dependencies:
+
+        ```json
+        {
+            "@spryker/frontend-config.tslint": "x.x.x",
+            "tslint": "x.x.x"
+        }
+        ```
+     
+    - Update the following commands:
+
+        ```json
+        {
+            "yves:tslint": "node ./frontend/libs/tslint",
+            "yves:tslint:fix": "node ./frontend/libs/tslint --fix"
+        }
+        ```
+      
+        Must be
+
+        ```json
+        {
+            "yves:eslint": "eslint --no-error-on-unmatched-pattern ./src/Pyz/Yves/**/Theme/**/*.{js,ts}",
+            "yves:eslint:fix": "eslint --no-error-on-unmatched-pattern --fix ./src/Pyz/Yves/**/Theme/**/*.{js,ts}"
+        }
+        ```
+
+2. Update and install package dependencies:
+
+```bash
+rm -rf node_modules
+npm install
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the `package-lock.json` file and the `node_modules` folder have been updated.
+
+{% endinfo_block %}
+
+## 2) Create or update configuration files
+
+1. Create `.eslintrc.json` file:
+
+```json
+{
+    "root": true,
+    "extends": ["./node_modules/@spryker/frontend-config.eslint/.eslintrc.js", "plugin:@typescript-eslint/recommended"],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2020,
+        "sourceType": "module",
+        "project": "./tsconfig.yves.json"
+    },
+    "plugins": ["deprecation"],
+    "ignorePatterns": [
+        "docker/",
+        "public/*/assets/",
+        "**/dist/",
+        "**/node_modules/",
+        "vendor/",
+        "src/Pyz/Zed/*/Presentation/Components/"
+    ],
+    "rules": {
+        "accessor-pairs": ["error", { "setWithoutGet": true, "enforceForClassMembers": false }],
+        "@typescript-eslint/no-empty-function": ["error", { "allow": ["methods"] }],
+        "@typescript-eslint/no-magic-numbers": [
+            "error",
+            {
+                "ignore": [-1, 0, 1],
+                "ignoreDefaultValues": true,
+                "ignoreClassFieldInitialValues": true,
+                "ignoreArrayIndexes": true,
+                "ignoreEnums": true,
+                "ignoreReadonlyClassProperties": true
+            }
+        ],
+        "deprecation/deprecation": "warn"
+    }
+}
+```
+
+2. Adjust `.github/workflows/ci.yml`:
```

Comment:
Suggest to unify this section, as CI might be different or even might be absent.